doc/lua_api.txt

```diff
@@ -2396,8 +2396,8 @@ Elements
 * `name` fieldname data is transferred to Lua
 * `caption 1`...: name shown on top of tab
 * `current_tab`: index of selected tab 1...
-* `transparent` (optional): show transparent
-* `draw_border` (optional): draw border
+* `transparent`: show transparent
+* `draw_border`: draw border
```

Comment:
Unrelated change. These arguments are optional.

Reply:
I don't know why I did this...